packages/sdk/src/client.ts

```diff
@@ -736,6 +746,51 @@ export function SoundClient({
     return network.chainId
   }
 
+  function _validateEditionConfig(config: EditionConfig) {
+    const { editionMaxMintableLower, editionMaxMintableUpper, fundingRecipient } = config
+
+    if (!isAddress(fundingRecipient) || fundingRecipient === '0x0000000000000000000000000000000000000000') {
```

Comment:
can we create a constants.ts and throw NULL_ADDRESS there?

Reply:
https://github.com/soundxyz/sdk/pull/133/commits/ecc5066876770376505d2d40480876ee0f444a35